datastore/schedule-export/main.py

```diff
@@ -3,8 +3,20 @@
 import os
 
 from googleapiclient.discovery import build
+from googleapiclient.discovery_cache.base import Cache
 
-datastore = build('datastore', 'v1')
+
+class MemoryCache(Cache):
+    _CACHE = {}
+
+    def get(self, url):
+        return MemoryCache._CACHE.get(url)
+
+    def set(self, url, content):
+        MemoryCache._CACHE[url] = content
+
+
+datastore = build('datastore', 'v1', cache=MemoryCache())
```

Comment:
Can we add a comment here to explain adding this?

```
Provide a `MemoryCache` instance as file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
```

Reply:
Have done, added a bit more to explain the "why" as well.